migration/orion-to-tessera/build.gradle

```diff
@@ -0,0 +1,173 @@
+    implementation project(":encryption:encryption-jnacl")
+    implementation project(":argon2")
+    implementation project(":key-generation")
+
+
+    implementation "org.bouncycastle:bcpkix-jdk15on:1.64"
+    implementation "org.bouncycastle:bcprov-jdk15on:1.64"
+
+    implementation "com.lmax:disruptor:3.4.2"
+
+
+    testImplementation "org.bouncycastle:bcpkix-jdk15on:1.64"
+    testImplementation "org.bouncycastle:bcprov-jdk15on:1.64"
+
+    implementation("net.consensys:orion:20.10.0") {
+        transitive = false
+    }
+    implementation "com.fasterxml.jackson.core:jackson-databind:2.12.2"
+    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.12.2"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.12.2"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-json-org:2.12.2"
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr353:2.12.2"
+
+    implementation "co.nstant.in:cbor:0.9"
+
+//    implementation("org.iq80.leveldb:leveldb-api:0.12")  {
+//        transitive = false
+//    }
+
+    implementation("org.iq80.leveldb:leveldb:0.12") {
+        transitive = false
+    }
+
+    implementation "org.glassfish:jakarta.json:1.1.6"
+
+    implementation "org.fusesource.leveldbjni:leveldbjni-all:1.8"
+    implementation "org.apache.tuweni:tuweni-bytes:1.1.0"
+    implementation "org.apache.tuweni:tuweni-crypto:1.1.0"
+    implementation "org.apache.tuweni:tuweni-config:1.1.0"
+
+    api "jakarta.validation:jakarta.validation-api:2.0.2"
+    implementation "org.hibernate.validator:hibernate-validator:6.1.6.Final"
+
+    runtimeOnly "org.eclipse.persistence:org.eclipse.persistence.jpa"
+    runtimeOnly "org.eclipse.persistence:org.eclipse.persistence.extension:"
+    api "jakarta.persistence:jakarta.persistence-api:2.2.3"
+
+    runtimeOnly group: 'org.apache.logging.log4j', name: 'log4j-slf4j18-impl', version: '2.14.0'
+
+    implementation "jakarta.xml.bind:jakarta.xml.bind-api:2.3.3"
+
+
+    implementation("org.eclipse.persistence:org.eclipse.persistence.moxy:2.7.7") {
+        exclude group: "jakarta.json", module: "jakarta,json-api"
+    }
+    implementation "org.glassfish.jaxb:jaxb-runtime:2.3.3"
+
+    testImplementation "com.h2database:h2:1.4.200"
+
+    implementation "com.zaxxer:HikariCP:3.2.0"
+
+}
+
+test {
+    systemProperty("javax.xml.bind.JAXBContextFactory", "org.eclipse.persistence.jaxb.JAXBContextFactory")
+    systemProperty("javax.xml.bind.context.factory", "org.eclipse.persistence.jaxb.JAXBContextFactory")
+    forkEvery = 1;
+    workingDir = "build/resources/test"
+
+    jacoco {
+        enabled = false //Skip coverage
+    }
+}
+
+
+
+task downloadAndUnpackSamples {
+    if(project.hasProperty("downloadOrionMigrationSamples")) {
+        ant.get(src: "https://codeload.github.com/ConsenSys/orion-to-tessera-migration-samples/zip/master", dest: "$buildDir/samples.zip", skipexisting: true)
```

Comment:
I haven't used codeload before so did some google-fu - is there a reason for using this url directly instead of api.gihub? https://stackoverflow.com/questions/60188254/how-is-codeload-github-com-different-to-api-github-com